README.md

```diff
@@ -46,12 +46,13 @@
 $ cargo add async-mongodb-session
 ```
 
-## Configuration
-
-This library utilises the document [expiry feature](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-after-a-specified-number-of-seconds) in mongodb to expire the session.
+## Overview
+This library allow you to utilises the document expiration feature based on a [specified number of seconds](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-after-a-specified-number-of-seconds) or in a [specific clock time](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-at-a-specific-clock-time) supported by mongodb to expire the session.
 
```

Comment:
I was thinking something along the lines of
"The **specified number of seconds** approach is designed to enable the session time out to be managed at the mongodb layer. This approach provides globally consistent session timeout across multiple processes but has the downside that all services using the same session collection **must** use the same timeout value.

The **specific clock time** clock time approach is where you require more flexibility on your session timeouts such as a different session timeout per running service or you would prefer to manage the session time out at the process level. This is more flexible but might lead to some perceived inconsistency in session timeout depending on your upgrade/rollout strategy."

Reply:
Hi @No9, Neat!! If you want I can add those lines. 👍

Reply:
@pepoviola If you're OK with it then lets add it in

Reply:
Hi @No9, added 👍 . Thanks for the help!